benchopt/utils/tests/test_parquet.py

```diff
@@ -0,0 +1,39 @@
+import json
+import tempfile
+import pandas as pd
+
+from benchopt.utils.parquet import to_parquet
+from benchopt.utils.parquet import get_metadata
+from benchopt.utils.parquet import update_metadata
+
+
+def test_parquet_metadata():
+    df = pd.DataFrame({
+        'a': range(4),
+        'b': list('abcd'),
+        'c': [.1, .2, .3, .4],
+    })
+
+    with tempfile.NamedTemporaryFile("rb+", suffix=".pq") as f:
+
+        path = f.name
+        metadata = {'test': 'info', 'plot_configs': [{}, {}, {}]}
+        to_parquet(df, path, metadata)
+
+        # Make sure the data is readable
+        df_new = pd.read_parquet(path)
+        assert (df_new == df).all().all()
```

Comment:
```suggestion
        assert df.equals(df_new)
```

I would remove it as the same check is done below

Reply:
The test needs to be done twice to verify updating the metadata did not change the file content